.circleci/base_config.yml

```diff
@@ -358,6 +412,10 @@ workflows:
           name: build-ce-x64
           preset: community-pr
           enterprise: false
+      - run-cppcheck:
+          name: cppcheck
+          requires:
+            - build-ce-x64
```

Comment:
EE should be used.

Reply:
@KVS85 this is still a draft PR

Reply:
We run for both as we now use project files, which can potential have different files compiled because of community and enterprise differences.

Reply:
There is always CE + EE for every single CE change (with the same base branch name in case it's not set explicitly because we ship both). If a cppcheck run for a CE branch doesn't show a problem and not stop a PR that doesn't mean EE cppcheck won't have it when an EE nightly or a package is built.